core/frontend/src/AccuDraw.ts

```diff
@@ -2916,6 +2920,7 @@ export class AccuDraw {
       (current.view.iModel === previous.view.iModel))
       return;
 
+    this._acsPickId = undefined;

```

Comment:
Is this one line the only thing required to fix picking decorations after changing files?

Reply:
Yes.

Reply:
Does the WIP stuff break anything? Are you expecting the WIP stuff to be reviewed too? Do you think maybe the one-line fix should be a separate PR?

Reply:
The WIP stuff is fully functional and replaces code that was removed from ui-test-app...I just plan to make additional changes based on the set of sketching tools in Gist.

The DynamicGraphicsProvider and how it's used in CreateLineStringTool is worth a look.